src/main/java/de/tum/in/www1/artemis/service/connectors/pyris/job/ExerciseChatJob.java

```diff
@@ -1,13 +1,16 @@
 package de.tum.in.www1.artemis.service.connectors.pyris.job;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+
 import de.tum.in.www1.artemis.domain.Course;
 import de.tum.in.www1.artemis.domain.Exercise;
 
 /**
- * An implementation of a PyrisJob for tutor chat messages.
- * This job is used to reference the details of a tutor chat session when Pyris sends a status update.
+ * An implementation of a PyrisJob for exercise chat messages.
+ * This job is used to reference the details of a exercise chat session when Pyris sends a status update.
  */
-public record TutorChatJob(String jobId, long courseId, long exerciseId, long sessionId) implements PyrisJob {
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public record ExerciseChatJob(String jobId, long courseId, long exerciseId, long sessionId) implements PyrisJob {
```

Comment:
Shouldn't DTOs be called ...DTO (ExerciseChatJobDTO)?

Reply:
@bassner